tests/e2e/specs/blocks/shopper/shopper-wc-blocks-saved-card-checkout-and-usage.spec.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable jest/no-test-prefixes */
+
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { shopper, merchant } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopperWCP,
+	merchantWCP,
+	describeif,
+	RUN_WC_BLOCKS_TESTS,
+	checkPageExists,
+} from '../../../utils';
+
+const billingDetails = config.get( 'addresses.customer.billing' );
+const productName = config.get( 'products.simple.name' );
+const card = config.get( 'cards.basic' );
+
+import { fillCardDetailsWCB } from '../../../utils/payments';
+
+describeif( RUN_WC_BLOCKS_TESTS )(
+	'WooCommerce Blocks > Saved cards with enabled UPE methods',
```

Comment:
Are these tests still for the enabled UPE methods? Tests look fine, but does this little description need a quick update?